pkg/deploy/api/helpers.go

```diff
@@ -17,3 +18,14 @@ func DeploymentToPodLogOptions(opts *DeploymentLogOptions) *kapi.PodLogOptions {
 		LimitBytes:   opts.LimitBytes,
 	}
 }
+
+// ScaleFromConfig builds a scale resource out of a deployment config.
+func ScaleFromConfig(dc *DeploymentConfig) *extensions.Scale {
+	return &extensions.Scale{
+		ObjectMeta: kapi.ObjectMeta{
+			Name:              dc.Name,
+			Namespace:         dc.Namespace,
+			CreationTimestamp: dc.CreationTimestamp,
```

Comment:
Since Scale objects don't have the normal object lifecycle (they're never persisted, they don't use a strategy, etc), I don't think so.